subprojects/docs/docs.gradle

```
@@ -550,6 +519,16 @@ samples {
             }
         }
 
+        publishingConventionPlugins {
+            sampleDirectory = samplesRoot.dir("build-organization/publishing-convention-plugins")
+            displayName = "Sharing configuration between projects in multi-repo setup"
+            description = "Organize and publish build configuration for reuse in other projects"
```

```suggestion
            description = "Organize and publish build logic for reuse in other projects"
```

I prefer _logic_ over _configuration, because the latter term is so overloaded (configuration time, configuration cache, configurations to work with dependencies, ...). Wdyt?